io.github.theforceengine.tfe.Mod.DarkForcesCommunityPatch.json

```diff
@@ -0,0 +1,43 @@
+{
+  "id": "io.github.theforceengine.tfe.Mod.DarkForcesCommunityPatch",
+  "runtime": "io.github.theforceengine.tfe",
+  "sdk": "org.freedesktop.Sdk//23.08",
+  "build-options": {
+    "strip": false,
+    "no-debuginfo": true
+  },
+  "separate-locales": false,
+  "appstream-compose": false,
```

Comment:
Please don't lecture me on building plugins. It will make you look bad.

Install `org.flatpak.Builder` and use that in lieu of `flatpak-builder`. And the do what's requested in the review.

As I said "it's no longer necessary". Involve using the latest flatpak-builder not a thre year old version or whatever your distro uses.

Reply:
never run it as root.

Reply:
Run this as your regular user, no need for sudo https://docs.flathub.org/docs/for-app-authors/submission#before-submission